pkgs/development/compilers/llvm/common/default.nix

```diff
@@ -577,9 +592,18 @@ let
             if stdenv.targetPlatform.libc == "llvm" then buildPackages.cmakeMinimal else buildPackages.cmake;
           python3 =
             if stdenv.targetPlatform.libc == "llvm" then
-              buildPackages.python3Minimal
+              # Prevent infinite-recursion within openssl
+              buildPackages.python3Minimal.override { openssl = null; }
```

Comment:
I was initially concerned by this, but this is fine, i.e. this is the intended way to invoke the python expression for building python without openssl for bootstrapping purposes.

Reply:
Yeah, I ran a build and it worked fine.